rust/crd/src/authentication.rs

```diff
@@ -130,6 +135,7 @@ pub struct AirflowClientAuthenticationDetailsResolved {
     pub user_registration: bool,
     pub user_registration_role: String,
     pub sync_roles_at: FlaskRolesSyncMoment,
+    pub tls_ca_cert_mount_path: Option<String>,
```

Comment:
It is not necessary to find a specific certificate because all of them are added to certifi.

```suggestion
```